src/encoded/tests/datafixtures.py

```diff
@@ -918,6 +918,17 @@ def pipeline_bam(testapp, lab, award, analysis_step_bam):
     return testapp.post_json('/pipeline', item).json['@graph'][0]
 
 
+@pytest.fixture
+def pipeline_noassay_bam(testapp, lab, award, analysis_step_bam):
```

Comment:
please rename the fixture to be something more clear, like "pipeline_without_assay_term_names"

Reply:
Ok, will implement a clearer name.